android/docs/help/about/adware.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Does Keyman include adware or spyware?
+---
+
+No!
+
+We never have and never will include adware or spyware components. 
+Keyman does contact the keyman.com website periodically to check for updates for both Keyman and the keyboards you have installed. 
```

Comment:
```suggestion
Keyman does contact the keyman.com website periodically to check for updates for the keyboards and predictive text dictionaries you have installed. 
```

Keyman updates are provided through the Play Store. Only keyboards/dicts are from keyman.com.

Can this be switched off in Settings? I can't remember, but if so we should document how here.